plugin/showmarks.vim

```diff
@@ -393,9 +393,12 @@ fun! s:ShowMarks()
 				endif
 				let mark_at{ln} = nm
 				if !exists('b:placed_'.nm) || b:placed_{nm} != ln
```

Comment:
Another way to fix this is to put an extra check here, [like this](https://github.com/vim-scripts/ShowMarks/pull/3).